pkg/controller/command/issuecredential/models.go

```diff
@@ -14,7 +14,7 @@ import "github.com/hyperledger/aries-framework-go/pkg/client/issuecredential"
 //
 type AcceptProposalArgs struct {
 	// PIID Protocol instance ID
-	PIID string `json:"piid"`
+	PIID string `json:"piid,omitempty"`
```

Comment:
why declare this as optional when we still have this check? https://github.com/hyperledger/aries-framework-go/blob/4a9b7fa924cd8aa3f218fb910eff42060cb01266/pkg/controller/command/issuecredential/command.go#L275-L278

Reply:
It allows reusing `AcceptProposalArgs ` structure. We are getting `piid` from the URL.

Reply:
That's not really RESTful, is it (then again, the whole API isn't...). If the ID is in the URI, then the more appropriate HTTP method would be `PUT` but it still remains `POST`. See [REST – PUT vs POST](https://restfulapi.net/rest-put-vs-post/).

Any reason why you can't set the PIID in the BDD tests?

Reply:
I see the piid on line 161 in `issuecredential_controller_steps.go`. This seems like an unnecessary change.

Reply:
Removed `omitempty`. 
`I see the piid on line 161 in issuecredential_controller_steps.go. This seems like an unnecessary change.` this change is necessary.

Reply:
@soluchok what I mean is the change to have the option of providing the piid in the URI is unnecessary. The PIID can be passed in the request body, and I don't see a reason why it isn't done so in the BDD tests.

Reply:
I am not sure about it. Currently, the URL looks like the following:
`/protocolname/{piid}/accept-proposal`
If we want to provide `piid` in the body we need to change the URL to
`/protocolname/accept-proposal`. 
Which actually affects everything. (unit/BDD/js tests, swagger etc.)
I cannot provide `piid` without those changes. 
If we really need it it can be done in a separate task.

Reply:
Oh I see :/

Reply:
How about `omitempty`? I would like to add it to every field. It will not affect anything.